authentication/template/src/App.test.jsx

```diff
@@ -14,16 +14,18 @@ jest.mock('@brightlayer-ui/react-auth-shared', () => ({
 
 test('renders welcome text', () => {
     render(
-        <ThemeProvider theme={createMuiTheme(BLUIThemes.blue)}>
-            <DrawerContext.Provider
-                value={{
-                    drawerOpen: true,
-                    setDrawerOpen: jest.fn(),
-                }}
-            >
-                <HomePage />
-            </DrawerContext.Provider>
-        </ThemeProvider>
+        <StyledEngineProvider injectFirst>
+            <ThemeProvider theme={createTheme(adaptV4Theme(BLUIThemes.blue))}>
```

Comment:
I don't think we need the v4 theme adapter if you use the latest alpha theme that was built for MUIv5